checkstyle-tester/projects-for-wercker.properties

```diff
@@ -7,6 +7,7 @@
 # Few projects that delivers set of unusual Java constructions that shall be correctly handled by AST visitor
 # 'InputAllEscapedUnicodeCharacters' must be skipped because it is too big and slows down JXR
 #checkstyle|git|https://github.com/checkstyle/checkstyle.git|master|**/checkstyle/src/test/resources-noncompilable/**/*,**/InputAllEscapedUnicodeCharacters.java
+#local-checkstyle|local|../../../|master|**/.ci-temp/**/*,**/resources-noncompilable/**/*,**/InputAllEscapedUnicodeCharacters.java
```

Comment:
would it make more sense to name this something like `local-checkstyle-ci` so it is clear it is for use by the CI only?
> `../../../`

Originally I was thinking the CI should create this line as we will be locking ourselves into this directory position from outside checkstyle's CI.

Reply:
I always look at them as execution on local ( the same command), CI just do this on our behalf.
Name of property file already imply CI.

Reply:
Sorry, didnt see this was wercker file. I was thinking this was the other one.